packages/liquidity-sdk-viem/src/loader.ts

```diff
@@ -205,26 +210,21 @@ export class LiquidityLoader<chain extends Chain = Chain> {
           ),
         });
 
-        const maxWithdrawalUtilization =
-          this.parameters.maxWithdrawalUtilization ??
-          fromEntries(
-            allVaultsMarkets.flatMap(([, markets]) =>
-              markets.map((market) => [
-                market.uniqueKey,
-                market.targetWithdrawUtilization,
-              ]),
-            ),
-          );
+        parameters.maxWithdrawalUtilization ??= fromEntries(
+          allVaultsMarkets.flatMap(([, markets]) =>
+            markets.map((market) => [
+              market.uniqueKey,
+              market.targetWithdrawUtilization,
+            ]),
+          ),
+        );
```

Comment:
This makes the constructor not pure so i'm not a huge fan :/ It might be okay in this specific case